layouts/default.vue

```diff
@@ -71,22 +71,24 @@
                 />
             </v-main>
         </div>
-        <div v-if="$store.state.priority.visualizationStage == 'stage2'">
+        <div v-if="$store.state.priority.visualizationStage == 'chart'">
             <v-main class="pa-0">
                 <AnalyticsPCDashboard />
             </v-main>
         </div>
-        <div v-if="$store.state.priority.visualizationStage == 'stage3'">
+        <div v-if="$store.state.priority.visualizationStage == 'tablePriority'">
             <v-main class="pa-0">
                 <PriorityTable />
             </v-main>
         </div>
-        <div v-if="$store.state.priority.visualizationStage == 'stage4'">
+        <div
+            v-if="$store.state.priority.visualizationStage == 'tableMonitoring'"
+        >
             <v-main class="pa-0">
                 <MonitoringTable />
             </v-main>
         </div>
-        <div v-if="$store.state.priority.visualizationStage == 'stage5'">
+        <div v-if="$store.state.priority.visualizationStage == 'tableLand'">
```

Comment:
Rename it to tableLandUse, following the pattern

Reply:
renamed